charms/trusty/clearwater-bono/config.yaml

```diff
@@ -24,6 +24,6 @@ options:
     description: Target request latency for overload control (microseconds)
     type: int
   repo:
-    default: http://repo.cw-ngv.com/juju-clearwater-1/
+    default: http://repo.cw-ngv.com/stable/
```

Comment:
Keep the default - otherwise this will cause bugs if people use it and our stable version has features which the Juju charm hasn't been updated to account for.